src/components/help-modal/index.css

```diff
@@ -65,4 +73,20 @@ kbd {
 .doc-image {
   height: 20px;
   margin: 0px 6px 0px 2px;
+  vertical-align: sub;
+}
+
+.tips {
+  text-decoration: none;
+  color: black;
```

Comment:
Why override it?

Reply:
Was overriding to make the link a button , removed